pom.xml

```diff
@@ -71,8 +71,8 @@
       <scope>provided</scope>
       <exclusions>
         <exclusion>
-          <groupId>org.apache.hadoop</groupId>
-          <artifactId>hadoop-client</artifactId>
+          <groupId>com.sun.jersey</groupId>
+          <artifactId>jersey-server</artifactId>
```

Comment:
- hadoop client brings in mapred which is needed to compile as PairRDDFunction has dependency on it. 
- jersey server exclusion is added because it bring in asm which collides with our dependencies and make addService() method fail during reflection.